webrtc.html

```diff
@@ -4306,6 +4307,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             <h2>Methods</h2>
             <dl data-link-for="RTCCertificate" data-dfn-for="RTCCertificate"
             class="methods">
+              <dt><dfn><code>getAlgorithms</code></dfn></dt>
+              <dd>
+                <p>Returns a sequence of recommended certificate algorithms.</p>
```

Comment:
On the "recommended", it seems that it might be better to say "supported" or "supported and recommended". Lets say we had some crappy hash that was no longer recommended by standards, but the browser still supported it. Would we return it? I don't care too much how we go on this but clearly we can't return things which are recommended by specs but not supported by browser.